lib/resources/http.rb

```diff
@@ -25,13 +25,14 @@ class Http < Inspec.resource(1)
     "
 
     # rubocop:disable ParameterLists
-    def initialize(url, method: 'GET', params: nil, auth: {}, headers: {}, data: nil)
+    def initialize(url, method: 'GET', params: nil, auth: {}, headers: {}, data: nil, ssl_verify: true)
       @url = url
       @method = method
       @params = params
       @auth = auth
       @headers = headers
       @data = data
+      @verify = ssl_verify
```

Comment:
Can we set the instance variable to `@ssl_verify` as well? `@verify` is fairly ambiguous, and all the other instance variables match the incoming arguments.